amundsen_application/api/search/v0.py

```diff
@@ -186,8 +186,8 @@ def _map_table_result(result: Dict) -> Dict:
             'cluster': result.get('cluster', None),
             'description': result.get('description', None),
             'database': result.get('database', None),
-            'schema_name': result.get('schema_name', None),
-            'last_updated_epoch': result.get('last_updated_epoch', None),
+            'schema': result.get('schema', None),
+            'last_updated_timestamp': result.get('last_updated_timestamp', None),
```

Comment:
In that case we would be better off renaming these to `epoch` instead of `timestamp`. I think there was another conversation elsewhere and we ended up going with `timestamp`. I don't recall seeing any non-epoch timestamps so far.